tt_metal/common/test_tiles.hpp

```diff
@@ -353,3 +334,24 @@ inline std::vector<T> convert_layout(
     }
     return std::vector<T>();
 }
+
+template <typename T, template <typename...> typename BufferType>
+inline std::vector<T> convert_layout(
+    const BufferType<T>& inp,
+    tt::stl::Span<const uint32_t> shape,
```

Comment:
should we pass SimpleShape here?

Reply:
I don't know if we want to introduce dependencies on ttnn in tt_metal code